tests/non_search/setup_helpers.py

```diff
@@ -147,6 +149,35 @@ def setup_with_two_docs(db: Session):
     )
 
 
+def setup_with_documents_large_with_families(
+    documents_large: list[Dict],
```

Comment:
Is the type here specific enough? I don't feel that strongly about this, but do we want:

 ```suggestion
    documents_large: list[Dict[str, Any]],
```

Reply:
Could get a bit carried away but technically I guess its something noisy like:
```
list[Dict[str, str | dict[str, str | list[str]]]]
```

The seems awful and extreme so maybe not that

Reply:
✅ I've gone with the first suggestions, as I'm not too fussed about type checking be wildly useful here for now.